### PR TITLE
Fix the Wonderland issues

### PR DESCRIPTION
**Pull request by manboy-eth**
Fixes the Zodiac app-related issues found in the Wonderland papers.

The issues are related to blockchain explorer configurations.

**Before merging**
We must add those environment variables to the production environment:

```
REACT_APP_ETHERSCAN_KEY=
REACT_APP_GOERLI_ETHERSCAN_KEY=
REACT_APP_GNOSISSCAN_KEY=
REACT_APP_POLYGONSCAN_KEY=
REACT_APP_BSCSCAN_KEY=
REACT_APP_OPTIMISTIC_ETHERSCAN_KEY=
REACT_APP_ARBISCAN_KEY=
REACT_APP_SNWOTRACE_KEY=
```

### CODE REVIEW

File: packages/app/.env.sample [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
+
+# Blockchain explorers
+REACT_APP_ETHERSCAN_KEY=
+REACT_APP_GOERLI_ETHERSCAN_KEY=
+REACT_APP_GNOSISSCAN_KEY=
+REACT_APP_POLYGONSCAN_KEY=
+REACT_APP_BSCSCAN_KEY=
+REACT_APP_OPTIMISTIC_ETHERSCAN_KEY=
+REACT_APP_ARBISCAN_KEY=
+REACT_APP_SNWOTRACE_KEY=
```
Comment (jfschwarz) [inline packages/app/.env.sample:15]: little typo here

```suggestion
REACT_APP_SNOWTRACE_KEY=
```

**Comment by jfschwarz** [comment]
We still have to add the new env vars the the GitHub workflow yaml files, reading from project secrets. For setting the project secrets admin access is required. I don't have it. Can one you add these, @samepant @auryn-macmillan ?

```
REACT_APP_ETHERSCAN_KEY=
REACT_APP_GOERLI_ETHERSCAN_KEY=
REACT_APP_GNOSISSCAN_KEY=
REACT_APP_POLYGONSCAN_KEY=
REACT_APP_BSCSCAN_KEY=
REACT_APP_OPTIMISTIC_ETHERSCAN_KEY=
REACT_APP_ARBISCAN_KEY=
REACT_APP_SNOWTRACE_KEY=
```

**Comment by auryn-macmillan** [comment]
@jfschwarz, I've gone ahead and added you as an admin

File: packages/app/.env.sample [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
 REACT_APP_INFURA_ID=
-REACT_APP_ETHERSCAN_KEY=
 REACT_APP_MULTI_SEND_CONTRACT=0xA238CBeb142c10Ef7Ad8442C6D1f9E89e07e7761
-REACT_APP_BACKEND_API_URL=http://localhost:3001/api
\ No newline at end of file
+REACT_APP_BACKEND_API_URL=http://localhost:3001/api
+
+# Blockchain explorers
+REACT_APP_ETHERSCAN_KEY=
+REACT_APP_GOERLI_ETHERSCAN_KEY=
```
Comment (jfschwarz) [inline packages/app/.env.sample:9]: For goerli we should use `REACT_APP_ETHERSCAN_KEY` (There's no separate profile)

**Comment by jfschwarz** [comment]
I've added the following project secrets:

image

**Review by auryn-macmillan — APPROVED** [review]
lgtm

**Pull request merged**